### ISSUE TITLE
NVDA doesn't read properly detail row's open button

### ISSUE THREAD

**Issue by RudolfDaniel** [i-Cell-Mobilsoft-Open-Source/icell-data-table#181]
**Describe the bug**
NVDA doesn't read properly detail row's open button.

**To Reproduce**
1. Start NVDA screen reader
2. Go to row what has expandable detail row
3. NVDA doesn't read state change of the expandable detail row (open/close)
4. NVDA reads out the svc icon as "figure"

**Expected behavior**
NVDA doesn't reads out "figure", but reads out the state change of the detail row.

**Desktop (please complete the following information):**
 - NVDA screen reader

**Comment by RudolfDaniel** [comment]
Fixed earlier.
